www/lib/utils.ts

```diff
@@ -0,0 +1,18 @@
+export function cmpBy<T>(...key: (keyof T)[]) {
```

Comment:
can we `utils.ts` -> `sorting.ts`? I have an aversion to generic names like `utils` and `common` and `helpers`

Reply:
Samesies. I had various pieces of code in here at various points of the refactor. `sorting` is good, will do.